db/migrate/20210702113010_assign_note_with_partner.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AssignNoteWithPartner < ActiveRecord::Migration[6.1]
+  def change
+    add_column :notes, :assign_with_partner, :boolean
```

Comment:
Why is this called `assign_with_partner` when it's about adding a note for a user?